src/features/patientsList/PatientsList.tsx

```diff
@@ -1,23 +1,85 @@
-import { Box, Theme } from '@material-ui/core';
+import { Box, Table, Theme, TableContainer, TableHead, TableRow, TableCell, TableBody, TablePagination } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
-import React, { FunctionComponent } from 'react';;
+import React, { FunctionComponent, useState } from 'react';;
 
 const useStyles = makeStyles((theme: Theme) => ({
 	navigationButton: {
 		padding: theme.spacing(3)
-    }
+	},
+	table: {
+		minWidth: 700,
+	},
+	container: {
+		height: '90vh',
+	}
 }));
 
+interface TableRowProps {
```

Comment:
JFYI: need to use more specific names, e.g. patient info or smth. else, as TableRowProps is very common